di_website/api/utils.py

```diff
@@ -19,9 +21,17 @@ def serialise_page(request, page, fields=['title', 'full_url', 'active']):
     """
     result = object_to_dict(page, fields)
     result['relative_url'] = page.relative_url(request.site, request)
-
     return result
 
+def serialiseDatasources(request, spotlightPage):
+    dataSourceLinks = []
+    for item in spotlightPage.datasources_links:
+        page = {}
+        page['caption'] = item.value['caption']
+        page['url'] = item.value['url']
+        page['page_url'] = item.value['page'].get_url(request, request.site) if item.value['page'] else ''
```

Comment:
We need only one of these, so it's an either/or situation. Just `url` is fine, with the hardcoded url taking precedent. If there's no url, check for a page. Basically, don't want this `page_url` vs `url` check being done on the React side.

Reply:
Updated this to be checked from the website